metricbeat/module/elasticsearch/ccr/_meta/docs.asciidoc

```diff
@@ -1,3 +1,8 @@
 This is the `ccr` metricset of the Elasticsearch module. It interrogates the
-Cross Cluster Replication Stats API endpoint to fetch information about shards
-in the Elasticsearch cluster that are participating in cross-cluster replication.
+Cross-Cluster Replication Stats API endpoint to fetch metrics about cross-cluster
+replication from the Elasticsearch clusters that are participating in cross-cluster
```

Comment:
Minor, but I'm trying to clean up the docs by using the correct attribute to resolve product names. So you should use {es} instead of Elasticsearch throughout this topic.